main.go

```diff
@@ -17,16 +19,39 @@ import (
 
 // Config contains the application configuration parameters
 type Config struct {
-	BaseDir      string   `envconfig:"BASE_DIR" default:"."`
-	Port         string   `envconfig:"PORT" default:"8000"`
-	AwsRegion    string   `envconfig:"AWS_REGION" default:"us-west-1"`
-	S3Bucket     string   `envconfig:"S3_BUCKET" default:"nitro-junk"`
-	ClusterSeeds []string `envconfig:"CLUSTER_SEEDS" default:"127.0.0.1"`
-	CacheSize    int      `envconfig:"CACHE_SIZE" default:"512"`
-	RedisPort    int      `envconfig:"REDIS_PORT" default:"6379"`
-	ClusterName  string   `envconfig:"CLUSTER_NAME" default:"default"`
+	BaseDir                 string   `envconfig:"BASE_DIR" default:"."`
+	Port                    string   `envconfig:"PORT" default:"8000"`
+	AwsRegion               string   `envconfig:"AWS_REGION" default:"us-west-1"`
+	S3Bucket                string   `envconfig:"S3_BUCKET" default:"nitro-junk"`
+	ClusterSeeds            []string `envconfig:"CLUSTER_SEEDS"`
```

Comment:
Note that I removed the default value from here, since I'm adding it below anyway.